package.json

```diff
@@ -31,18 +31,22 @@
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
+    "@embroider/macros": "^1.16.5",
     "ember-cli-babel": "^7.26.10",
     "ember-cli-htmlbars": "^5.7.2"
   },
+  "peerDependencies": {
+    "ember-data": "^3.16.0 || ^4.0.0 || ^5.0.0"
```

Comment:
The peerDep is required to make the dependencySatisfies macro work as expected for some reason.

_technically_ it could be considered a breaking change, but since the range is so wide I don't think it is.